build/VSIX.targets

```diff
@@ -44,7 +32,8 @@
       <VSIXOutputPath>$(BuildDir)$(VSIXName).vsix</VSIXOutputPath>
     </PropertyGroup>
 
-    <CallTarget Targets="_RestoreProject" />
+    <Exec Command="&quot;$(MSBuildExePath)&quot; &quot;$(VSIXProject)&quot; /nologo /t:Restore /v:m" />
```

Comment:
Yup. PackageReference in old-school csproj should work too once you have the latest public 15.3 preview. I didn't change to that yet, though, as this caused issues for the VSIX targets.